Source/santactl/Commands/SNTCommandMetrics.m

```diff
@@ -68,14 +68,33 @@ - (void)prettyPrintMetricValues:(NSDictionary *)metrics {
 
     for (NSString *fieldName in metric[@"fields"]) {
       for (NSDictionary *field in metric[@"fields"][fieldName]) {
-        const char *fieldNameStr = [fieldName cStringUsingEncoding:NSUTF8StringEncoding];
-        const char *fieldValueStr = [field[@"value"] cStringUsingEncoding:NSUTF8StringEncoding];
+        //const char *fieldNameStr = [fieldName cStringUsingEncoding:NSUTF8StringEncoding];
+        //const char *fieldValueStr = [field[@"value"] cStringUsingEncoding:NSUTF8StringEncoding];
         const char *createdStr = [field[@"created"] UTF8String];
         const char *lastUpdatedStr = [field[@"last_updated"] UTF8String];
         const char *data = [[NSString stringWithFormat:@"%@", field[@"data"]] UTF8String];
 
-        if (strlen(fieldNameStr) > 0) {
-          printf("  %-25s | %s=%s\n", "Field", fieldNameStr, fieldValueStr);
+        NSArray<NSString *> *fields = [fieldName componentsSeparatedByString:@","];
+        NSArray<NSString *> *fieldValues = [field[@"value"] componentsSeparatedByString:@","];
+
+        if (fields.count != fieldValues.count) {
+          //TODO Log Error
```

Comment:
Why todo? Can LOGE not be used?

Reply:
This is a user facing so probably best to print to stderr instead no? Otherwise we need to add logging as a dependency to the command.

Reply:
fwiw, `LOGE` will print to stderr for things like `santactl` ran from the terminal